api/src/lib/auth.js

```diff
@@ -12,41 +12,38 @@ import { matrix } from 'src/lib/roles'
 export const getCurrentUser = async (decoded, { token, type }) => {
   try {
     let user = await db.user.findUnique({
-      where: { email: decoded.preferred_username }
+      where: { email: decoded.preferred_username },
+      include: {
+        UserRole: true
+      }
     })
-    //if no user found... create one
     let justRoles = []
     if (user) {
-      let roles = await db.userRole.findMany({
-        where: { userId: user.id }
-      })
-      justRoles = roles.map((role) => {
+      justRoles = user.UserRole.map((role) => {
         return role.name
```

Comment:
because roles were included above, they don't need to be called a second time here.